### PR TITLE
Experiment to make body - `display:block`

### PR DESCRIPTION
**Pull request by sriramkrish85**
https://github.com/ampproject/amphtml/issues/5310

### CODE REVIEW

File: src/style-installer.js [hunk 1]
```diff
@@ -110,12 +111,20 @@ export function insertStyleElement(doc, cssRoot, cssText, isRuntimeCss, ext) {
...
+  const win = doc.defaultView;
+  const shouldMakeBodyBlock = isExperimentOn(win, 'make-body-block');
+  const bodyStyles = {
+    opacity: 1,
+    visibility: 'visible',
+    animation: 'none',
+  };
+
+  if (shouldMakeBodyBlock) {
+    bodyStyles['display'] = 'block';
```
Comment (dvoytenko) [inline src/style-installer.js]: This is unfortunately too late for this. Let's do this in `viewport-impl.js` on top level somewhere.

File: tools/experiments/experiments.js [hunk 2]
```diff
@@ -177,6 +177,11 @@ const EXPERIMENTS = [
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/4715',
   },
   {
+    id: 'make-body-block',
+    name: 'Sets the body to display:block.',
+    spec: 'https://github.com/ampproject/amphtml/issues/5310',
```
Comment (dvoytenko) [inline tools/experiments/experiments.js]: Please add `cleanupIssue`.

File: src/style-installer.js (hunk 1 above)
Comment (sriramkrish85) [inline src/style-installer.js]: done.

File: tools/experiments/experiments.js (hunk 2 above)
Comment (sriramkrish85) [inline tools/experiments/experiments.js]: Done

File: css/amp.css [hunk 3]
```diff
@@ -47,6 +47,11 @@ body {
   text-size-adjust: 100%;
 }
 
+/** make-body-block experiment */
+.make-body-block body {
```
Comment (dvoytenko) [inline css/amp.css]: This has to be a `-amp-make-body-block` class.

File: src/service/viewport-impl.js [hunk 4]
```diff
@@ -168,6 +168,15 @@ export class Viewport {
...
       setStyle(this.globalDoc_.documentElement, 'touch-action', 'pan-y');
     }
+
+    // TODO(sriramkrish85, #5319): Cleanup the experiment by making the effects
+    // on CSS permanent and removing the code block below.
+    if (this.ampdoc.isSingleDoc() &&
+            isExperimentOn(this.ampdoc.win, 'make-body-block')) {
+      this.globalDoc_.documentElement.classList.add('make-body-block');
+    }
+
```
Comment (dvoytenko) [inline src/service/viewport-impl.js]: nit: extra lines

File: css/amp.css (hunk 3 above)
Comment (sriramkrish85) [inline css/amp.css]: Done

File: src/service/viewport-impl.js (hunk 4 above)
Comment (sriramkrish85) [inline src/service/viewport-impl.js]: Done

**Pull request merged**